src/main/scala/li/cil/oc/integration/enderstorage/DriverFrequencyOwner.java

```diff
@@ -84,9 +84,10 @@ public Object[] setFrequency(final Context context, final Arguments args) {
             return null;
         }
 
-        @Callback(doc = "function():string -- Get the name of the owner, which is usually a player's name or 'global'.")
+        @Callback(doc = "function():string or nil -- Get the name of the owner, which is usually a player's name or nil.")
```

Comment:
Keeping compatibility with older code would be nice - so it might be worth returning "global" here, instead of null.

Reply:
Except it has been returning null in 1.12. Well then... @payonel - I'd say we fix the 1.12 docs to document the nil behaviour.